packages/ui/src/views/repo/repo-settings/components/repo-settings-general-security.tsx

```diff
@@ -97,6 +111,26 @@ export const RepoSettingsSecurityForm: FC<RepoSettingsSecurityFormProps> = ({
             <Message theme={MessageTheme.ERROR}>{errors.secretScanning.message?.toString()}</Message>
           )}
 
+          {isMFE ? (
```

Comment:
i'd prefer this to be a more specific condition.. eg the prop could be `showVulnerabilityScanning` instead.
not a blocker.